PackageExplorer/App.xaml.cs

```diff
@@ -25,6 +25,7 @@ public partial class App : Application
         public App()
 #pragma warning restore CS8618 // Non-nullable field is uninitialized.
         {
+            AppContext.SetSwitch("System.Net.Http.UseSocketsHttpHandler", false);
```

Comment:
I prefer to have a comment in code for this decision. Otherwise the reason could be hard to find in the future.

Reply:
Thanks!